src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -133,9 +136,9 @@ public interface UserRepository extends JpaRepository<User, Long> {
      *
      * @param page        Pageable related info (e.g. for page size)
      * @param loginOrName Either a login (e.g. ga12abc) or name (e.g. Max Mustermann) by which to search
-     * @return            list of found users that match the search criteria
+     * @return list of found users that match the search criteria
```

Comment:
I think this formatting was correct, see e.g. this change: https://github.com/ls1intum/Artemis/pull/5163/files#r882969773

Reply:
Reverted - thanks to my autoformater :P